bazel/README.md

```diff
@@ -417,5 +417,55 @@ a [JSON Compilation Database](https://clang.llvm.org/docs/JSONCompilationDatabas
 with any tools (e.g. clang-tidy) compatible with the format.
 
 The compilation database could also be used to setup editors with cross reference, code completion.
-For example, you can use [You Complete Me](https://valloric.github.io/YouCompleteMe/) or 
+For example, you can use [You Complete Me](https://valloric.github.io/YouCompleteMe/) or
 [cquery](https://github.com/cquery-project/cquery) with supported editors.
+
+# Advanced caching setup
+
+Below is the advanced caching setup, it will make bazel cache works efficiently when you run
+different compilation mode and/or have multiple working tree.
+
+## Setup local HTTP cache
+
+```
+go run github.com/buchgr/bazel-remote --dir ${HOME}/bazel_cache --host 127.0.0.1 --port 28080 --max_size 64
```

Comment:
This did not work for me. It might be because of some google-specific restrictions, setup, or lack thereof. This is what I get:
```
package https:/github.com/buchgr/bazel-remote: cannot find package "https:/github.com/buchgr/bazel-remote" in any of:
	/usr/lib/google-golang/src/https:/github.com/buchgr/bazel-remote (from $GOROOT)
	/usr/local/google/home/jmarantz/go/src/https:/github.com/buchgr/bazel-remote (from $GOPATH)
```
I don't have `$GOPATH` or `$GOROOT` set, and 'go' resolves to `/usr/lib/google-golang/bin/go`. I'm not sure if that would be different on a vanilla ubuntu distribution.

I tried also git cloning the package but that did not work for me either. I have to admit that while I've done quite a bit of Go, but only under blaze, so I'm not quite sure what I'm doing wrong. Maybe more explicit instructions would help.

Reply:
I think it is internal version of Go doesn't allow you to download and run arbitrary package :) Trying with vanilla go1.11 on ubuntu (GCE) and it worked fine for me.